File: algoExpert/easy/bubbleSort.js
function bubbleSort(array) {
  let sortCount = false;

  for (let i = 0; i < array.length; i++) {
    if (array[i] > array[i + 1]) {
      const temp = array[i];
      array[i] = array[i + 1];
      array[i + 1] = temp;
      sortCount = true;
    }
  }
  if (sortCount) {
    bubbleSort(array);
  }
  return array;
}

File: algoExpert/python/easy/bubbleSort.py
def bubbleSort(array):
	isSorted = False

	while not isSorted:
		isSorted = True
		for i in range(len(array)-1):
			if array[i] > array[i+1]:
				swap(i, i+1, array)
				isSorted = False
	return array

def swap(i, j, array):
	array[i], array[j] = array[j], array[i]
